app/src/main/java/com/github/onlynotesswent/ui/common/ImageComposables.kt

```diff
@@ -112,33 +111,28 @@ fun NonModifiableProfilePicture(
     fileViewModel: FileViewModel,
     size: Int = 150,
     testTag: String = "profilePicture",
-    onClick: () -> Unit = doesNothing
+    onClick: (() -> Unit)? = null
```

Comment:
Replacing onClick to be nullable and removing the doesNothing makes the code cleaner, nice.